Sources/TSCBasic/misc.swift

```diff
@@ -162,8 +162,8 @@ public func exec(path: String, args: [String]) throws -> Never {
     guard execv(path, cArgs.cArray) != -1 else {
         throw SystemError.exec(errno, path: path, args: args)
     }
-    fatalError("unreachable")
   #endif
+    fatalError("unreachable")
```

Comment:
This shouldn't follow on Windows due to the use of `_exit`.

Reply:
Ah, I see @neonichu opened another PR (#285), let's go with that one.